### DISCUSSION TITLE
docker+superset for mysql

### DISCUSSION THREAD

**Discussion by EnsonY** [apache/superset#14522] [Q&A / Help] [answered]
Add **mysql** database to **superset**

I use the Windows system, install Docker according to the official document instructions, and install SuperSet based on Docker. I cannot connect to MySQL now, do not know how to install the driver。

**Comment by EnsonY** [comment]
superset_init occurs errors ，but i don't think it would cause mysql disconnect to superset
![image](https://user-images.githubusercontent.com/19368974/117435444-746db480-af60-11eb-948d-05f1810a0640.png)
![image](https://user-images.githubusercontent.com/19368974/117435476-7df71c80-af60-11eb-96dd-cb3124d95657.png)
![image](https://user-images.githubusercontent.com/19368974/117435602-9d8e4500-af60-11eb-8227-d951bece397c.png)

**Reply by srinify** [reply]
@EnsonY FYI I have answered this in the Superset Slack

**Comment by EnsonY** [comment, marked as answer]
> Add **mysql** database to **superset**
> 
> I use the Windows system, install Docker according to the official document instructions, and install SuperSet based on Docker. I cannot connect to MySQL now, do not know how to install the driver。

just change the localhost to the ip mapped to docker
